Best Practices Docs Update Branch (#13887)
microsoft/fluentui: packages/office-ui-fabric-react/src/components/Dropdown/docs/DropdownOverview.md

> A drop-down menu (`Dropdown`) is a list in which the selected item is always visible, and the others are visible on demand by clicking a drop-down button.

Reviewer: ```suggestion
A drop-down menu (`Dropdown`) is a list in which the selected item is always visible, while the non-selected items are visible on demand by clicking a drop-down button.
```

Reviewer 2: I didn't go with your exact suggestion, because "non-selected" felt a bit too technical. So, I edited this to, "...the selected item is always visible while other items are visible on demand by selecting a dropdown button."
